scim-spec/scim-spec-schema/src/main/java/org/apache/directory/scim/spec/filter/InMemoryScimFilterMatcher.java

```diff
@@ -87,9 +87,11 @@ protected Predicate<R> unhandledExpression(FilterExpression expression, Attribut
     return scimResource -> false;
   }
 
-  private static abstract class AbstractAttributePredicate<T extends FilterExpression, R> implements Predicate<R> {
+  public <A> A get(Schema.Attribute attribute, A actual) {
+    return attribute.getAccessor().get(actual);
+  }
 
-    private static final Logger log = LoggerFactory.getLogger(InMemoryScimFilterMatcher.class);
+  private abstract class AbstractAttributePredicate<T extends FilterExpression, R> implements Predicate<R> {
```

Comment:
This diff doesn't render well via github, but it's adding a new method and removing `final` from `AbstractAttributePredicate`